src/SFA.DAS.Commitments.Application.UnitTests/Commands/UpdateApprenticeshipStatus/WhenStoppingAnAwaitingApprenticeship.cs

```diff
@@ -168,7 +167,7 @@ public async Task ThenShouldCallTheRepositoryToUpdateTheStatus()
             MockApprenticeshipRespository.Verify(x => x.StopApprenticeship(
                 It.Is<long>(a => a == 123L),
                 It.Is<long>(a => a == ExampleValidRequest.ApprenticeshipId),
-                It.Is<DateTime>(a => a == ExampleValidRequest.DateOfChange)));
+                It.Is<DateTime>(a => a == ExampleValidRequest.DateOfChange),It.IsAny<bool?>()));
```

Comment:
append as new line to match existing formatting

Reply:
done